### ISSUE TITLE
REMOVE SQUID

### ISSUE THREAD

**Issue by konecat** [n3wm1nd/TA#776]
TLL sonar jammer is overpowered and should be removed, unless subs have underwater LOS implemented.

**Comment by skymyj** [comment]
Maybe just delete Sonar Jammer.

**Comment by playerO1** [comment]
If you do "just delete Sonar Jammer" you delete Squid unit.
Maybe just modified sonar jammer? Just increase jammer range. And, add/increase decloak area. But increase jammer and forgot add decloack area will be too good for TLL he-he.
So, TLL not have advanced generator on water - it do unbalance. The sonar jammer add new chance to win for TLL.
The anti-minefield weapon Juno can shut town at the short time the sonar jammer.
All water mine do not work vs sonar jamming units.

**Comment by konecat** [comment]
Balance for you is making a unit that can't be countered? Please... just stop

also TLL is a faction that is WIP so it's normal things are missing.

**Comment by skymyj** [comment]
Why not transform this unit into spy sub ? 
We delete the jammer sonar. Add the Stealth & Cloak option & EMP autodestroy ?

**Comment by nixtux** [comment]
Yeah there is no los underwater just sonar, and no I won't remove this unit

**Comment by skymyj** [comment]
Ok. But what do you want for this unit ? 
Just keep the radar jammer ? Or transform it into spy ?

**Comment by nixtux** [comment]
Will stay the same, I'm working on something

**Comment by nixtux** [comment]
Ok won't stay, needs fixing started a gadget but it would just a hax mess, this need addressing again
